src/main/java/io/spring/issuebot/Repository.java

```diff
@@ -86,14 +86,11 @@ else if (!this.name.equals(other.name)) {
 			return false;
 		}
 		if (this.organization == null) {
-			if (other.organization != null) {
-				return false;
-			}
+			return other.organization == null;
 		}
-		else if (!this.organization.equals(other.organization)) {
-			return false;
+		else {
```

Comment:
This could just `return this.organization.equals(other.organization);` rather than having the `else` block, couldn't it?

Reply:
Yes, fixed